server/src/main/java/org/apache/iotdb/db/http/handler/GetTimeSeriesHandler.java

```diff
@@ -0,0 +1,76 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.http.handler;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import org.apache.iotdb.db.auth.AuthException;
+import org.apache.iotdb.db.auth.AuthorityChecker;
+import org.apache.iotdb.db.exception.StorageEngineException;
+import org.apache.iotdb.db.exception.metadata.MetadataException;
+import org.apache.iotdb.db.exception.query.QueryProcessException;
+import org.apache.iotdb.db.http.constant.HttpConstant;
+import org.apache.iotdb.db.metadata.PartialPath;
+import org.apache.iotdb.db.qp.physical.sys.ShowTimeSeriesPlan;
+import org.apache.iotdb.db.query.context.QueryContext;
+import org.apache.iotdb.db.query.control.QueryResourceManager;
+import org.apache.iotdb.tsfile.exception.filter.QueryFilterOptimizationException;
+import org.apache.iotdb.tsfile.read.common.Field;
+import org.apache.iotdb.tsfile.read.common.RowRecord;
+import org.apache.iotdb.tsfile.read.query.dataset.QueryDataSet;
+import org.apache.thrift.TException;
+
+public class GetTimeSeriesHandler extends Handler {
+  public JsonElement handle(JsonArray json)
+      throws AuthException, MetadataException, TException, StorageEngineException,
+      QueryFilterOptimizationException, IOException, InterruptedException, SQLException, QueryProcessException {
+    checkLogin();
+    JsonArray result = new JsonArray();
+    for(JsonElement object : json) {
+      String path = object.getAsString();
+      PartialPath partialPath = new PartialPath(path);
+      ShowTimeSeriesPlan plan = new ShowTimeSeriesPlan(partialPath);
+      plan.setHasLimit(true);
+      if(!AuthorityChecker.check(username, plan.getPaths(), plan.getOperatorType(), null)) {
+        throw new AuthException(String.format("%s can't be gotten by %s", path, username));
+      }
+      long queryID = QueryResourceManager.getInstance().assignQueryId(false);
+      QueryDataSet dataSet = executor.processQuery(plan, new QueryContext(queryID));
+      while(dataSet.hasNext()) {
+        JsonArray row = new JsonArray();
+        RowRecord rowRecord = dataSet.next();
+        List<Field> fields = rowRecord.getFields();
+        for(Field field : fields) {
+          if(field != null) {
+            row.add(field.getStringValue());
+          } else {
+            row.add(HttpConstant.NULL);
+          }
+        }
+        result.add(row);
+      }
+    }
```

Comment:
Please keep the json format for result. like
{
    "timeseries": "root.sg.d1.s1",
    "alias": "temperature",
    ...
}.
It will be more usable. Otherwise I have to remember the column index, if i want to get one specific column.